docs/topic/timeseries/learn/with-metadata.md

```
@@ -50,6 +54,12 @@ like manufacturer, model, and firmware version.
 
 ## Creating the Tables
 
+:::{div}
+The fastest and easiest way to get started with CrateDB is by deploying a
+free tier (CRFREE) cluster on [CrateDB Cloud][CrateDB Cloud Console]. Otherwise,
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "cratedb cloud console"

(MD052, reference-links-images)